common/src/test/java/org/apache/falcon/entity/parser/FeedEntityParserTest.java

```diff
@@ -1160,6 +1160,24 @@ public void testImportFeedSqoopInvalid() throws Exception {
         Assert.fail("ValidationException should have been thrown");
     }
 
+    @Test (expectedExceptions = {ValidationException.class, FalconException.class})
+    public void testImportFeedWithNoTimePartition() throws Exception {
+
+        InputStream feedStream = this.getClass()
+                .getResourceAsStream("/config/feed/feed-import-no-timepartition-0.1.xml");
+        parser.parseAndValidate(feedStream);
+        Assert.fail("ValidationException should have been thrown");
```

Comment:
Hey @vramachan Thanks for fixing the docs and expected exceptions. Actually in this statement I meant something else. I am curious about the need for that Assert.fail statement. Even in failure scenarios, even without this statement we will get clear message that the ValidationException was expected but was not thrown. Anyhow, it is just nit picking :stuck_out_tongue: